app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -245,7 +246,13 @@ protected void onResumeFragments() {
             return;
         }
 
-        tryAutoLogin();
+        if (CommCareApplication._().isConsumerApp()) {
+            uiController.setUsername("t1");
+            uiController.setPasswordOrPin("123");
+            localLoginOrPullAndLogin(false);

```

Comment:
..this is just me completely forgetting to do this properly so thank you for flagging